ce/api/geo.py

```diff
@@ -124,5 +125,14 @@ def polygonToMask(nc, fname, poly, variable):
             raise Exception("Unable to determine projection parameters for GDAL "
                             "dataset {}".format(dst_name))
 
-        mask = rasterize((poly,), out_shape=raster.shape, transform=raster.affine, all_touched=True)
```

Comment:
Is the `raster.affine` check a leftover from the old raster masking? It looks like it. Could it be removed?